src/main/java/com/commercetools/sync/services/impl/ProductServiceImpl.java

```diff
@@ -20,9 +18,8 @@
 import java.util.Optional;
 import java.util.Set;
 import java.util.concurrent.CompletionStage;
-import java.util.stream.Collectors;
 
-import static java.lang.String.format;
+import static com.commercetools.sync.commons.utils.CtpQueryUtils.buildResourceKeysQueryPredicate;
```

Comment:
Honestly, I am not seeing any reason to change the ProductService also the need for making the function common, maybe you could explain ?

Reply:
My intension was to re-use in several areas, but as im using built-in predicates from the JVM SDK now i reverted this change.